src/sonic-config-engine/minigraph.py

```diff
@@ -157,11 +161,94 @@ def parse_png(png, hname):
                             elif node.tag == str(QName(ns, "EndDevice")):
                                 mgmt_dev = node.text
 
-    return (neighbors, devices, console_dev, console_port, mgmt_dev, mgmt_port, port_speeds, console_ports)
+    return (neighbors, devices, console_dev, console_port, mgmt_dev, mgmt_port, port_speeds, console_ports, device_neighbors)
```

Comment:
i do not understand why need to instroduce device_neighbors here. can we just use the neighbors here?

Reply:
Removed usage of device_neighbors from parse_png